docs/api/sources.md

```diff
@@ -0,0 +1,137 @@
+# Sources endpoint
+
+Insert blurb describing what sources are.
+
+## Getting started
+
+The endpoints used in this guide are part of the agenda Agenda APIs. Before continuing, please review the [getting started guide](./getting-started.md) for important information that you need to know in order to successfully make calls to the API, including required headers and how to read example APIs.
+
+## List sources
+
+You can retrieve a list of all the sources for your company by making a GET request to the `/sources` endpoint.
+
+### API format
+
+The `/sources` endpoint supports several query parameters to help filter your results. While these parameters are optional, their usage is **strongly** recommended to help reduce expensive overhead. Making a call to this endpoint without any parameters will retrieve **all** sources available for your company. 
+
+```http
+GET /sources
+GET /sources?category={CATEGORY}
+GET /sources?type={TYPE}
+GET /sources?limit={LIMIT}
+GET /sources?page={PAGE}
+```
+
+| Parameter | Description |
+| --------- | ----------- |
+| `{CATEGORY}` | The category of source you want to filter by. This value can either be `virtual` or `in-person`. |
+| `{TYPE}` | The type of source you want to filter by. This value 
+is **only** used by `virtual` sources. Possible values include `zoom`, `slack`, `teams`, and `skype`. |
+| `{LIMIT}` | Specifies the number of sources returned. | 
+| `{PAGE}` | Specifies the offset of the pages of sources returned. |
+
+### Request
+
+The following request will retrieve the last two created sources for the company.
+
+```shell
+curl -X GET https://api.agenda.com/sources?limit=2 \
+-H 'Authorization: Bearer {ACCESS_TOKEN}' \
+-H 'Client-ID: {API_KEY}'
+```
+
+### Response
+
+A successful response returns HTTP status 200 with a list of sources based on the query parameter provided in the request path.
+
+```json
+{
+    "records": [
+        {
+            "name": "Solis",
+            "category": "in-person",
+            "id": 8,
+            "createdBy": 1,
+            "createdAt": 1639498959,
+            "updatedAt": 1639498959
```

Comment:
Should `in-person` sources have a field that provides basic info as to where it is? Which office, which room, etc? 

Now that I'm thinking about it, calling an IRL meeting room a "source" isn't exactly intuitive, even if it makes sense for virtual meetings. What if we call these "rooms" instead? A room can be a virtual room (whose source can be Slack, Teams, etc.), or it can be a physical room.

Reply:
My brain was thinking that the name would encompass the idea of like location - but you're right, probably having an explicit location parameter makes more sense, (so it can be like Solis, which is 3/F Room 317, for example).